src/lib.rs

```diff
@@ -128,9 +181,9 @@ impl<'a> Iterator for SMRecIterator<'a> {
 
         // StkMapRecord[NumRecords] {
         //     uint64: PatchPoint ID
-        let sm_id = itry!(cursor.read_u64::<NativeEndian>());
+        let id = itry!(cursor.read_u64::<NativeEndian>());
```

Comment:
Was `sm_id` misleading?

Reply:
Not misleading, but for consistency, I thought it was better to mirror the names of fields on the structs that we create.